docs/using-latest-daily.md

```diff
@@ -37,7 +37,7 @@ If you use [Package Source Mapping](https://learn.microsoft.com/en-us/nuget/cons
 To be able to create aspire projects, you will need to install the latest Aspire templates. You can do this by running the following command:
 
 ```sh
-dotnet new install Aspire.ProjectTemplates::9.0.0-preview.* --force
+dotnet new install Aspire.ProjectTemplates::9.0.0-rc.* --force
```

Comment:
```suggestion
dotnet new install Aspire.ProjectTemplates::9.0.0-.* --force
```

Reply:
Also Aspire.ProjectTemplates.net8 for the .NET 8 version of the templates

Reply:
If you are dogfooding, who wants net8?

Reply:
Actually, I'm going to roll this back. This will start grabbing builds from main, which I don't think we want

Reply:
split into 2 sections. first is for "latest daily". In a TIP section it says how to install rc1